subprojects/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.9-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.0.2-all.zip
```

Comment:
7.1?

Reply:
There was a problem with change Copy task behavior;
If multiple `into` specified, only first one actually copied

Reply:
If I'm not mistaken, this behavior was already in 6.9.